### ISSUE TITLE
How to define role / permission structure

### ISSUE THREAD

**Issue by cord** [ZiNETHQ/spark-roles#12]
trying to get Spark Roles working but somehow I can't figure out how to define the roles and permission structure. 

In the wiki are some examples like `canatleast(['edit.blog.post', 'create.blog.post']` but how do I define the permissions and map them to roles?

**Comment by mikestratton** [comment]
I agree that this would be helpful. Otherwise it might be better to just use Laravel's default authorization methods by writing Gates: https://laravel.com/docs/5.7/authorization

**Comment by drjonnicholson** [comment]
So you can create instances of the `Permission` model (https://github.com/ZiNETHQ/spark-roles/blob/master/src/Models/Permission.php) and then use it's ID to assign it to a role, e.g. for a permission with ID 1 `$role->assignPermission(1);`

Hope that helps
